internal/engine/wazevo/backend/isa/arm64/instr.go

```diff
@@ -785,10 +793,18 @@ func (i *instruction) asMovFromVec(rd, rn operand, arr vecArrangement, index vec
 	i.u1, i.u2 = uint64(arr), uint64(index)
 }
 
-func (i *instruction) asVecDup(rd, rn operand, arr vecArrangement) {
+func (i *instruction) asMovFromVecSignedFIXME(rd, rn operand, arr vecArrangement, index vecIndex) {
```

Comment:
the silly name is because I'd like to merge this with `asMovFromVec`, but it was easier to make it work with some copy/paste first